Packs/iDefense/Integrations/iDefense_v2/iDefense_v2.py

```diff
@@ -0,0 +1,274 @@
+
+    if len(analysis_info):
+        readable_output = tableToMarkdown('Results', analysis_info)
+        indicator = Common.URL(url, dbot)
+    else:
+        readable_output = f"No results were found for url {url}"
+
+    return CommandResults(indicator=indicator,
+                          raw_response=res,
+                          readable_output=readable_output)
+
+
+def domain_command(client: Client, args: dict) -> CommandResults:
+    domain: str = str(args.get('domain'))
+    res = client.perform_request('/domain', domain)
+    analysis_info, dbot = _extract_analysis_info(res, domain, DBotScoreType.DOMAIN)
+
+    indicator: Optional[Common.Domain] = None
+
+    if len(analysis_info):
+        readable_output = tableToMarkdown('Results', analysis_info)
+        indicator = Common.Domain(domain, dbot)
+
+    else:
+        readable_output = f"No results were found for domain {domain}"
+
+    return CommandResults(indicator=indicator,
+                          raw_response=res,
+                          readable_output=readable_output)
+
+
+def uuid_command(client: Client, args: dict) -> CommandResults:
+    """
+    Search for indicator with the given uuid. When response return, checks which indicator found.
+    Args:
+        client: iDefense client
+        args: arguments obtained with the command representing the value to search
+
+    Returns:
+        CommandResults containing the indicator, the response and a readable output
+    """
+    uuid: str = str(args.get('uuid'))
+    try:
+        res = client.perform_request(f'/{uuid}', '')
+    except Exception as e:
+        if 'Failed to parse json object from response' in e.args[0]:
+            return CommandResults(indicator=None, raw_response={},
+                                  readable_output=f"No results were found for uuid {uuid}")
+        else:
+            raise e
+
+    dbot_score = _calculate_dbot_score(res.get('severity'))
+    desc = 'Match found in IDefense database'
+
+    indicator_value = res.get('key')
+    indicator_type = res.get('type')
+    indicator: Optional[Union[Common.IP, Common.Domain, Common.URL]] = None
+    # Create indicator by the uuid type returned
+    if indicator_type.lower() == 'ip':
+        dbot = Common.DBotScore(indicator_value, DBotScoreType.IP, 'iDefense', dbot_score, desc)
+        indicator = Common.IP(indicator_value, dbot)
+    elif indicator_type.lower() == 'domain':
+        dbot = Common.DBotScore(indicator_value, DBotScoreType.DOMAIN, 'iDefense', dbot_score, desc)
+        indicator = Common.Domain(indicator_value, dbot)
+    elif indicator_type.lower() == 'url':
+        dbot = Common.DBotScore(indicator_value, DBotScoreType.URL, 'iDefense', dbot_score, desc)
+        indicator = Common.URL(indicator_value, dbot)
+
+    analysis_info = {
+        'Name': res.get('key'),
+        'Dbot Reputation': dbot_score,
+        'confidence': res.get('confidence'),
+        'Threat Types': res.get('threat_types')
```

Comment:
better not to use space in the context keys

Reply:
Yes, Dean also told me about it. But in the old integration they return md with spaces, so he told me its better to keep it the same. What do you think?

Reply:
the command name is different `uuid` -> `idefense-get-ioc-by-uuid`
so as we change the old behaviour anyhow, we should change the output paths as well.

Reply:
Only in uuid or on all commands?

Reply:
let's fix in all of them.